common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -676,10 +685,9 @@ public static int equalsConstantTime(byte[] bytes1, int startPos1, byte[] bytes2
      * The resulting hash code will be case insensitive.
      */
     public static int hashCodeAscii(byte[] bytes, int startPos, int length) {
-        if (!hasUnsafe() || !unalignedAccess()) {
-            return hashCodeAsciiSafe(bytes, startPos, length);
-        }
-        return PlatformDependent0.hashCodeAscii(bytes, startPos, length);
+        return !hasUnsafe() || !PlatformDependent0.unalignedAccess() ?
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Unnecessary use of fully qualified name 'PlatformDependent0.unalignedAccess' due to existing static import 'io.netty.util.internal.PlatformDependent0.unalignedAccess' [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUnnecessaryFullyQualifiedName)